src/config/routes/2.1.json

```diff
@@ -1,5 +1,11 @@
 [
     {
+        "path": "/events(/[^/]+)/talks/?$",
```

Comment:
Should you be catching `\d+` rather than `[^/]+`? Events have numeric IDs; it's probably wise to add a bit of extra sanity